virttest/utils_disk.py

```diff
@@ -601,6 +603,165 @@ def create_filesystem(session, partition_name, fstype, ostype, timeout=360):
         create_filesyetem_linux(session, partition_name, fstype, timeout)
 
 
+def get_mpoint_fstype_linux(session, partition):
```

Comment:
I think you want `get_fs_attrs_linux()`, otherwise, it should be a private function if you just want some of the attrs.

Reply:
thanks, updated~